truco-front/src/gameLogic/type/GameAction.ts

```diff
@@ -47,4 +47,21 @@ export const isEnvidoAction = (action: GameAction): boolean => {
 
 export const isTurnAction = (action: GameAction): boolean => {
     return action === GameAction.PLACE_CARD || action === GameAction.IR_AL_MAZO
+}
+
+export const getPrintableAction = (action: GameAction): string => {

```

Comment:
i like this, maybe we can add a few variations to every response

Reply:
Done